### ISSUE TITLE
Right-click on any link or action button to open it in a new browser tab

### ISSUE THREAD

**Issue by FreeUKGenIssues** [FreeUKGen/FreeUKRegProductIssues#1253]
Issue reported by **SBManager** at 2019-11-08 14:27:37 UTC
Time: 2019-11-08T10:32:23+00:00
Session ID: 717d6b2f1b867028a1d832be9ba7c2a2
Problem Page URL: [/manage_resources/new?locale=en](/manage_resources/new?locale=en)
Previous Page URL: [https://www.freereg.org.uk/cms/refinery/login](https://www.freereg.org.uk/cms/refinery/login)
Reported Issue:
Action button links that take you away from the main FeeREG site such as Roadmap and Refinery should open in a new browser tab so that the FR site remains readily usable.    

![Screenshot](https://www.freereg.org.uk)

E0 U2 S3 C6: P11

**Comment by edickens** [comment]
Creating replies to Contact, Feedback and Messages would also be useful in a separate tab so that it is possible to look up information at the same time as composing the reply.

**Comment by PatReynolds** [comment]
Members to be told (underneath actions) to use right click where they wish to open in a new tab for internal links (including refinery), and external links should always open in a new tab.

@edickens to add to newletter

**Comment by stsccfr** [comment]
Right-clicking on something that follows a link to open a (browser-controlled) menu that allows opening in a new tab or window is now quite conventional and doesn't really need to be documented.
